metadata-ingestion/source_docs/bigquery.md

```diff
@@ -0,0 +1,111 @@
+```
+
+## Config details
+
+Note that a `.` is used to denote nested fields in the YAML recipe.
+
+| Field                  | Required | Default                                                        | Description                                                                                                                                                                                                                                                                                                                                                                            |
+| ---------------------- | -------- | -------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `projects`             |          |                                                                |                                                                                                                                                                                                                                                                                                                                                                                        |
+| `extra_client_options` |          |                                                                |                                                                                                                                                                                                                                                                                                                                                                                        |
+| `env`                  |          | `"PROD"`                                                       | Environment to use in namespace when constructing URNs.                                                                                                                                                                                                                                                                                                                                |
+| `bucket_duration`      |          | `"DAY"`                                                        | Duration to bucket usage events by. Can be `"DAY"` or `"HOUR"`.                                                                                                                                                                                                                                                                                                                        |
```

Comment:
Why would a user need to care about this..?

Reply:
We should probably remove this from the example config above. It's confusing. We should just say, "By default, we extract usage stats for the last day, with the recommendation that this source is executed every day."